cartopy/documentation.html

```diff
@@ -53,14 +54,32 @@
           <h2>Previous releases</h2>
             <ul>
                 <li><a href="docs/v0.13/index.html">v0.13</a></li>
+            </ul>
```

Comment:
Good point - in that case it has brought this page in line with Iris' equivalent. This suggests that it is coming from the template file, which can be edited to rationalise the lists into a single list. Either way, I inherited the code, but I should probably fix it nonetheless